README.md

```diff
@@ -57,6 +80,18 @@ const resolverFunc = (parent, args, context, info) => { }
 
 const [gql, resolver] = usecase2query(usecase, resolverFunc)
 ```
+If you put custom name or custom name or convention:
+```javascript
+//Custom name
```

Comment:
this explanation is repeating in many parts of the readme, I believe we can create a section to this feature and just create some reference here, what do you think?

Reply:
well observed, in the readme was separated into a specific session